lib/keila/templates/html_text_extractor.ex

```diff
@@ -0,0 +1,140 @@
+defmodule Keila.Templates.HtmlTextExtractor do
+  @doc """
+  Processes an HTML fragment tree and returns it as a string.
+  """
+  @spec extract_text(Keila.Templates.Html.t(), String.t()) :: String.t()
+  def extract_text(html, selector \\ "body") do
+    Floki.find(html, selector)
+    |> do_extract_text(0)
+    |> concat_extracted_text()
+    |> then(fn string -> string <> "\n" end)
```

Comment:
In this line, if you change to this:
`<> "\n"`

It will have the same result, no?

Reply:
It would work without the then, but then the `<> "\n"` wouldn’t be part of the pipe anymore and it would mess up the formatting :sweat_smile:

Reply:
You are right 😅